atd-vzd/schema/fatalities.sql

```diff
@@ -0,0 +1,122 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 14.6 (Debian 14.6-1.pgdg110+1)
+-- Dumped by pg_dump version 14.6 (Ubuntu 14.6-0ubuntu0.22.04.1)
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- Name: fatalities; Type: TABLE; Schema: public; Owner: visionzero
+--
+
+CREATE TABLE public.fatalities (
+    id integer NOT NULL,
+    crash_id integer,
+    location text,
+    victim_name text,
+    law_enforcement_num integer,
+    ytd_fatal_crash integer,
+    ytd_fatality integer,
+    person_id integer,
+    primaryperson_id integer,
+    year text
+);
```

Comment:
If each row must have either a `person_id` or `primaryperson_id` (and not both?), you could consider adding a [check constraint](https://www.postgresql.org/docs/9.0/ddl-constraints.html#AEN2385) that will enforce that rule. 

A `comment` on the table describing it's purpose would be a nice touch, too.

Reply:
ah yeah thats a great point, i will add that!

Reply:
and correct its either not both

Reply:
just want to point out that this table is already in staging/prod. so it will need to be dropped and re-added and re-tracked in hasura.

also, this sql is missing the `is_deleted` column.